### ISSUE TITLE
 Val_loss Calculation with Multiple Validation Sets in Sliding Window Technique and Early Stopping

### ISSUE THREAD

**Issue by rd1886** [jdb78/pytorch-forecasting#1547]
Hi,

I am using the sliding window technique for training, along with early stopping based on validation loss. Given that this approach results in multiple validation sets, I'm wondering how the val_loss used for early stopping is calculated per epoch. Is it the average of the validation losses across all validation sets?

Thanks!
